### PR TITLE
Expandtests

### PR DESCRIPTION
**Pull request by saulakravitz**
- Expand tests based on examples from IG
- Fix way that RestoreReferences finds the Certifier

### CODE REVIEW

File: VRDR.Tests/DeathRecord_Should.cs [hunk 1]
```diff
@@ -987,6 +1032,8 @@ public void Get_GivenNames()
             Assert.Equal("Middle", ((DeathRecord)JSONRecords[0]).GivenNames[1]);
             Assert.Equal("Madelyn", ((DeathRecord)XMLRecords[0]).GivenNames[0]);
             Assert.Equal("Middle", ((DeathRecord)XMLRecords[0]).GivenNames[1]);
+            Assert.Equal("Madelyn", ((DeathRecord)JSONRecords[2]).GivenNames[0]);
+            Assert.Equal(1, ((DeathRecord)JSONRecords[2]).GivenNames.Length);
```
Comment (pkmitre) [inline VRDR.Tests/DeathRecord_Should.cs:1036]: Need to address the warnings you see when you run the tests (in this case not using Assert.Equal(...), using Assert.Single(((DeathRecord)JSONRecords[2]).GivenNames)

File: VRDR/DeathRecord.cs [hunk 2]
```diff
@@ -10161,15 +10144,20 @@ private void RestoreReferences()
...
-            (entry.Resource.Id != null && entry.Resource.Id == Composition.Attester.First().Party.Reference)));
-            if (practitionerEntry != null)
+            if ( Composition == null || (Composition.Attester == null || Composition.Attester.FirstOrDefault() == null || Composition.Attester.First().Party == null || String.IsNullOrWhiteSpace(Composition.Attester.First().Party.Reference)))
             {
-                Certifier = (Practitioner)practitionerEntry.Resource;
+                if(fullRecord)
+                {
+                    throw new System.ArgumentException("The Composition is missing an attestor (a reference to the Certifier/Practitioner resource).");
+                }
+            }else{  // There is an attester
```
Comment (pkmitre) [inline VRDR/DeathRecord.cs:10153]: Need to use correct formatting (brackets on their own line)

Comment (pkmitre) [reply VRDR/DeathRecord.cs:10153]: (Just run the formatter)

File: VRDR/IJEMortality.cs [hunk 3]
```diff
@@ -610,6 +610,8 @@ private string Dictionary_Geo_Get(string ijeFieldName, string fhirFieldName, str
                     {
                         current = "N";
                     }
+                } else if (geoType == "countyC" || geoType == "cityC"){
```
Comment (pkmitre) [inline VRDR/IJEMortality.cs:613]: Need to run the formatter...

**Review by pkmitre — CHANGES_REQUESTED** [review]

File: VRDR.Tests/DeathRecord_Should.cs (hunk 1 above)
Comment (saulakravitz) [reply VRDR.Tests/DeathRecord_Should.cs]: I had trouble with this one.   See lines 562-564 of DeathRecord_Should.cs....

File: VRDR/DeathRecord.cs (hunk 2 above)
Comment (saulakravitz) [reply VRDR/DeathRecord.cs]: option-shift-f, option-shift-f, option-shift-f.... my new mantra.

**Review by pkmitre — APPROVED** [review]

**Pull request merged**